### ISSUE TITLE
source doesn't exists : app/templates/.gitignore

### ISSUE THREAD

**Issue by Toilal** [skatejs/generator-skatejs#7]
```
events.js:160
      throw er; // Unhandled 'error' event
      ^
AssertionError: Trying to copy from a source that does not exist: /home/toilal/.config/yarn/global/node_modules/generator-skatejs/generators/app/templates/.gitignore
    at EditionInterface.exports.copy (/home/toilal/.config/yarn/global/node_modules/mem-fs-editor/lib/actions/copy.js:47:3)
    at _moveToProject (/home/toilal/.config/yarn/global/node_modules/generator-skatejs/generators/app/index.js:116:13)
    at writing (/home/toilal/.config/yarn/global/node_modules/generator-skatejs/generators/app/index.js:93:10)
    at Object.<anonymous> (/home/toilal/.config/yarn/global/node_modules/yeoman-generator/lib/index.js:417:23)
    at /home/toilal/.config/yarn/global/node_modules/yeoman-generator/node_modules/run-async/index.js:25:25
    at /home/toilal/.config/yarn/global/node_modules/yeoman-generator/node_modules/run-async/index.js:24:19
    at /home/toilal/.config/yarn/global/node_modules/yeoman-generator/lib/index.js:418:9
    at runCallback (timers.js:651:20)
    at tryOnImmediate (timers.js:624:5)
    at processImmediate [as _immediateCallback] (timers.js:596:5)
```

**Comment by Toilal** [comment]
This file seems to be missing from npm package : https://unpkg.com/generator-skatejs@0.2.0/generators/app/templates/.gitignore

**Comment by Toilal** [comment]
Workaround : `yarn global add https://github.com/alexlafroscia/generator-skatejs`

**Comment by alexlafroscia** [comment]
Hmmm, that's interesting! Thanks for catching that, I'll try to figure out how to fix it. Probably just need to add a custom `.npmignore` file.

**Comment by alexlafroscia** [comment]
Just released version `0.3.0` that should fix this issue. Please let me know!
